Rakefile

```diff
@@ -1,16 +1,19 @@
 require "bundler/gem_tasks"
 require "cucumber/rake/task"
 require "rspec/core/rake_task"
-require "rubocop/rake_task"
```

Comment:
We don't install rubocop on all versions of Ruby, because some versions don't work. So we can't assume rubocop will be present. I have wrapped this in a begin/end to handle that case.